Makefile

```diff
@@ -46,7 +46,7 @@ check: $(LNT) $(CHK) $(DUP)
 nofea=$(strip $(foreach f,Initials Keyboard Mono,$(findstring $f,$1)))
 
 $(BUILDDIR)/%.ff.otf: $(SOURCEDIR)/%.sfd $(GSUB) $(BUILD)
-	echo "      BUILD  $(*F)"
+	$(info =>     BUILD  $(*F))
```

Comment:
My VIM configuration highlights NBSP and I input it often that is usually accessible from my keyboard layout, so I’d rather use it than these faux arrows.

Reply:
Roger that.